npa.js

```diff
@@ -17,6 +17,7 @@ let os
 const isWindows = process.platform === 'win32' || global.FAKE_WINDOWS
 const hasSlashes = isWindows ? /\\|[/]/ : /[/]/
 const isURL = /^(?:git[+])?[a-z]+:/i
+const isGit = /^git@[^:]+:/i
```

Comment:
what about https://www.npmjs.com/package/git ?

Reply:
oohh nvm haven't seen the rest of the regex there 😅

Reply:
Hmm...

Given that there is a package named `git`, you could have `git@npm:fork-of-git`, which seems like it'd match this regexp inappropriately?

Also, what about a git repo that doesn't use `git` as the username?  `isaacs@izs.me:some/repo.git` for example.

Reply:
yeah, it really relies on the `:`, which thankfully is an invalid character for a tag so we won't accidentally think a tag is a git repo. you did just highlight an edge case though, where `git@file:../my-git` or `git@npm:notreallygit` or even `git@github:npm/git` will fail, so still a good catch!